packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -7,8 +7,12 @@ import fs from "fs";
 import { assert, bufferToString } from "@fluidframework/common-utils";
 import {
     IDocumentStorageService,
-    IDocumentStorageServicePolicies,    // these are needed for api-extractor
-    ISummaryContext,                    // these are needed for api-extractor
+
+    /* eslint-disable @typescript-eslint/no-unused-vars */
+    // these are needed for api-extractor
+    IDocumentStorageServicePolicies,
```

Comment:
This frequently means that some function below returns one as a value?  In which case it's probably preferable to add an explicit return type to that function to make it not-unused

Reply:
I'll double check. I had thought the same initially, but ended up leaving this as it was for a reason I don't remember now...